docs/concepts/models/overview.md

```diff
@@ -227,10 +227,84 @@ Learn more about these properties and their default values in the [model configu
+
+    If [`cron`](#cron) is specified with a cron expression, however, SQLMesh uses a more complex approach to derive the `interval_unit`.
+
+    A [cron expression](https://en.wikipedia.org/wiki/Cron) can generate complex time intervals, so SQLMesh does not parse it directly. Instead, it:
+
+      1. Generates the next five run times from the cron expression (relative to the time of calculation)
+      2. Calculates the duration of the intervals between those five values
+      3. Determines the model's `interval_unit` as the largest interval unit value that is less than or equal to the minimum duration from (2)
+
+    For example, consider a cron expression corresponding to "run every 43 minutes." Its `interval_unit` is `half_hour` because that is the largest `interval_unit` value *shorter* than 43 minutes. If the cron expression is "run every 67 minutes", its `interval_unit` is `hour` given the same logic.
+
+    However, `interval_unit` does not have to be inferred from [`cron`](#cron) - you can specify it explicitly to customize how your backfill occurs.
+
+    #### Specifying `interval_unit`
+
+    Models often run on a regular cadence, where the same amount of time passes between each run and the same time length of data is processed in each run.
+
+    For example, a model might run at midnight every day (1 run per day) to process the previous day's data (1 day's worth of data per run). The length of time between runs and the time length of data processed in each run are both 1 day (or both 2 days if you miss a run).
+
+    However, the run cadence length and processed data length do not have to be the same.
+
+    Consider a model that runs every day at 7:30am and processes data up until 7am today. The model's `cron` is a cron expression for "run every day at 7:30am," from which SQLMesh infers an `interval_unit` of `day`.
+
+    What will happen when this model runs? First, SQLMesh will identify the most recent completed interval. The `interval_unit` was inferred to be `day`, so the last complete interval was yesterday. SQLMesh will not include any of today's data between 12:00am and 7:00am in the run.
+
+    To include today's data, manually specify an `interval_unit` of `hour`. When the model runs at 7:30am, SQLMesh will identify the most recent completed `hour` interval as 6:00-7:00am and include data through that interval in the backfill.
+
+    ```sql
+    MODEL (
+        name sqlmesh_example.up_until_7,
+        kind INCREMENTAL_BY_TIME_RANGE (
+          time_column date_column,
+        ),
+        start '2024-11-01',
+        cron '30 7 * * *', -- cron expression for "every day at 7:30am"
+        interval_unit 'hour', -- backfill up until the most recently completed hour (rather than day)
+      );
+    ```
+    <br></br>
+    **NOTE: the example below is an advanced use case. Use the `allow_partials` configuration option with caution.**
```

Comment:
allow partials is bad, can elaborate more 

https://tobikodata.com/data-completeness.html

1. you don't know if missing data is due to a partial run, or if there was a problem
2. if you ever miss running things, you could end up with incomplete data forever

in generally, we recommend not doing allow_partials